modules/gdscript/language_server/gdscript_extend_parser.cpp

```diff
@@ -269,7 +269,32 @@ void ExtendGDScriptParser::parse_class_symbol(const GDScriptParser::ClassNode *p
 					if (j > 0) {
 						symbol.detail += ", ";
 					}
-					symbol.detail += m.signal->parameters[j]->identifier->name;
+					const ParameterNode &this_parameter = *m.signal->parameters[j];
+					// signal_arg_name
+					symbol.detail += this_parameter.identifier->name;
+					// todo: What to do with parameters without a given type?
+					// todo: What to do with parameters of non-built-in type?
+					// has signal_arg_type?
```

Comment:
```suggestion
					symbol.detail += this_parameter.identifier->name;
```

Ask questions at the appropriated places, but not inside the engine code, please.